public/lib.js

```diff
@@ -265,6 +265,17 @@ function addImageByUrl(
      imageContainer.classList.remove('expanded')
     } else {
      imageContainer.classList.add('expanded')
+     imageContainer.focus()
+    }
+   },
+   keydown({ key }) {
```

Comment:
We can remove this event handler as the global `addEventListener` will handle the Escape key press